docs/framework/includes/managed-identities.md

```diff
@@ -0,0 +1,2 @@
+> [!NOTE]
+> Microsoft recommends that you use the most secure authentication flow available. If you're connecting to Azure SQL, [Managed Identities for Azure resources](/sql/connect/ado-net/sql/azure-active-directory-authentication#using-managed-identity-authentication) is the recommended authentication method.
```

Comment:
Here's the new include file.

Reply:
I will create the same include for aspnetcore.docs and aspnetdocs by the same name.  Thanks for leading the way!